Image uses appropriate scales

andreiggr opened the pull request:
- no longer loading the full image, but loading large scale from plone, `wide` is going to get `huge` scale
- images have width/height set, small req by lighthouse
- image will use scales provided by plone
- small performance improvement

ichim-david commented on src/customizations/volto/components/manage/Blocks/Image/Edit.jsx:
```diff
@@ -196,213 +147,206 @@ class Edit extends Component {
-   * @returns {undefined}
-   */
-  onKeyDownVariantMenuForm = (e) => {
+  // /**
```
@andreiggr the code was already commented fine there is no reason to have inline comments

avoinea commented:
@andreiggr Can you please rebase this branch on develop and fix the conflicting merge files?

andreiggr commented:
Fixed image scales selector after rebase with develop changes

Also fixed an align for the image input, when there is no image url, but the align remains on the container
Before.
![input bad align](https://user-images.githubusercontent.com/44702393/230353974-11bdeb95-2aa9-4617-be61-620f41b85e71.png)

After.
![input good align if no img](https://user-images.githubusercontent.com/44702393/230354201-f396644a-df39-4391-aef0-863a205170d2.png)
